common/README.md

```diff
@@ -0,0 +1,2 @@
+# geo engine common
+Common functionality for the other geo engine crates
```

Comment:
Is there a way to (auto) generate a config description?

Reply:
you mean generate a sample config from the config.get() calls? I dont't think so